components/script/dom/webglbuffer.rs

```diff
@@ -42,8 +42,8 @@ impl DroppableField {
         assert!(self.is_deleted());
         let cmd = WebGLCommand::DeleteBuffer(self.id);
         match operation_fallibility {
-            Operation::Fallible => self.context.send_command_ignored(cmd),
-            Operation::Infallible => self.context.send_command(cmd),
+            Operation::Fallible => self.sender.send(cmd, capture_webgl_backtrace()),
+            Operation::Infallible => self.sender.send(cmd, capture_webgl_backtrace()).unwrap(),
```

Comment:
Should `capture_webgl_backtrace()` take `WebGLRenderingContext`?
Is there a way to get `WebGLRenderingContext` from `WebGLMessageSender`?

Reply:
No. Instead, we add these helpers and call them instead:
```rust
#[cfg(not(feature = "webgl_backtrace"))]
#[inline]
pub fn stub_webgl_backtrace() -> WebGLCommandBacktrace {
    WebGLCommandBacktrace {}
}

#[cfg(feature = "webgl_backtrace")]
pub fn stub_webgl_backtrace() -> WebGLCommandBacktrace {
      WebGLCommandBacktrace {
          backtrace: String::new(),
          js_backtrace: None,
      }
    }
}
```